pkg/agent/config.go

```diff
@@ -12,14 +12,22 @@ import (
 )
 
 type Config struct {
+	RefreshStateIntervalSeconds uint `json:"refereshStateIntervalSeconds"`
+
 	Scaling   ScalingConfig    `json:"scaling"`
 	Metrics   MetricsConfig    `json:"metrics"`
 	Scheduler SchedulerConfig  `json:"scheduler"`
 	Monitor   MonitorConfig    `json:"monitor"`
+	NeonVM    NeonVMConfig     `json:"neonvm"`
 	Billing   *billing.Config  `json:"billing,omitempty"`
 	DumpState *DumpStateConfig `json:"dumpState"`
 }
 
+type RateThresholdConfig struct {
+	IntervalSeconds uint `json:"intervalSeconds"`
+	Threshold       uint `json:"threshold"`
```

Comment:
I noticed there's validation for `IntervalSeconds` but not `Threshold`, is that intentional?

Reply:
I'd expect 0 to be a valid threshold, meaning infinity.

Reply:
It doesn't mean infinity. It means `> 0`.